docker/Dockerfile.compile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:xenial
+
+RUN apt-get update && apt-get install -y \
+  cmake make \
+  clang g++ \
+  libboost-iostreams-dev libboost-system-dev \
+  libgmp-dev \
+  liblua5.2-dev libluajit-5.1-dev \
```

Comment:
To me it is actually inverse, if I compile with LuaJIT installed it ignores Lua52